reverseBits.py

```diff
@@ -0,0 +1,9 @@
+class Solution:
+    # @param n, an integer
+    # @return an integer
+    def reverseBits(self, n):
+        ans = 0
+        for i in range(32):
+            ans = (ans << 1) + (n & 1)
```

Comment:
Why do you need `()`?

Reply:
if no (), the output for 1 will be zero. but the correct answer should be   2147483648 (10000000000000000000000000000000).